content/_releases/23/08.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Product Update - v23.08
+description: Details of the product update v23.08 from 27th of February 2023.
+layout: rel
+version: "23.08"
+section: Product Updates
+category: product-updates
+releaseDate: 2023-02-27
+redirect_from:
+  - /releases/23.08.html
+---
+
+## Features
+
+### Configurable Recipe Use for Contracts
+
+As a continuation of the updates introduced in Release [23.06](/releases/23/06#new-recipe-createedit-ui-features), we have added new features in recipe management. This feature allows you to control which users are allowed to create/edit Recipes (by default, all users can use the recipes feature). You can set it up with a new permission `tenants.contract.edit_feature_flags` - allow to set the feature flags for Contract.
```

Comment:
```suggestion
As a continuation of the updates introduced in Release [23.06](/releases/23/06#new-recipe-createedit-ui-features), we have added new features in recipe management. This feature allows Tenant Admins and Owners to control which users are allowed to create/edit Recipes (by default, all users can use the recipes feature). You can set it up with a new permission `tenants.contract.edit_feature_flags` - allow to set the feature flags for Contract.
```

This feature is only available to certain tenant-level Users - lets please make that obvious